src/components/Layout/Layout.tsx

```diff
@@ -313,7 +287,7 @@ export const Layout = ({
                     )}
 
                     <div className="layout-sidebar-menu">
-                      {menu}
+                      <Menu currentPlatform={currentPlatform} path={asPathWithNoHash} />
```

Comment:
Just passing in the route and current platform into the menu and letting it figure out which menu should be rendered there